Examples/Scripts/Python/ckf_tracks.py

```diff
@@ -2,13 +2,81 @@
 from pathlib import Path
 from typing import Optional, Union
 from collections import namedtuple
+import argparse
+import sys
+import os
 
 from acts.examples import Sequencer, GenericDetector, RootParticleReader
 
 import acts
 
 from acts import UnitConstants as u
 
+def getArgumentParser():
```

Comment:
The point of the python binding is not to have parsers in them but to serve as example that people can modify. Could you remove your changes to this file ?

Reply:
@andiwand and @timadye since you are the one that usually work on the python binding part maybe you can comment ?

Reply:
Yes, we decided to keep the example scripts as simple examples, and not to complicate any more than we can help. That makes it easy to understand what's required and let the user customize.

Can you leave the defaults unspecified, and only add settings required to improve the performance? I think it's also clearer to add the settings you are changing directly in the call to `SeedfinderConfigArg`, rather than pass them in from `runCKFTracks`.

Of course you can put these tuning options into your own customized edit of this script. I don't know if that's useful to include in the acts repo. Perhaps more useful is to keep them in a personal repo, eg. as I did for [`full_chain_itk_test.py`](https://gitlab.cern.ch/adye/acts-setup/-/blob/master/acts-local/Examples/Scripts/Python/full_chain_itk_test.py).